src/components/common/wallet-button/index.js

```diff
@@ -0,0 +1,48 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+import * as ReactRouter from 'react-router'
+import Coinbase from 'components/common/icon/coinbase'
+import Status from 'components/common/icon/status'
+import Trust from 'components/common/icon/trust'
+
+import style from './index.style.js'
+
+class WalletButton extends React.Component {
+  render() {
+    const { targetWallet } = this.props
+    return (
+      <a css={style.base} href={this.link(targetWallet)}>
+        {this.wallet(targetWallet)}
+      </a>
+    )
+  }
+
+  wallet = targetWallet => {
+    switch (targetWallet) {
+      case 'Status':
+        return <Status />
+      case 'Trust':
+        return <Trust />
+      case 'Coinbase':
+        return <Coinbase />
+      default:
+        return null
+    }
+  }
+
+  link = targetWallet => {
+    switch (targetWallet) {
+      case 'Status':
+        return 'https://dev.status.im/'
+      case 'Trust':
+        return 'https://trustwallet.com/'
+      case 'Coinbase':
+        return 'https://wallet.coinbase.com/'
+      default:
+        return null
+    }
+  }
```

Comment:
Use constants instead:
```js
// constants/wallet_button.js

const TYPE = {
  STATUS: 'Status',
  TRUST: 'Trust',
  COINBASE: 'Coinbase'
}
const TYPES = Object.values(TYPE)

const LINK = {
  [TYPE.STATUS]: '...',
  ...
}

const ICON = {
  [TYPE.STATUS]: Icon.Status,
  ...
}

const COLOR = {
  [TYPE.STATUS]: STYLE.COLOR.STATUS,
}
```
```js
// wallet-button/index.js

render() {
  const { type } = this.props

  return (
	<ActioinButton
	  link={WALLET_BUTTON.LINK[type]
	  ...
	/>
  )
}
```
```js
...
render = () =>
  WALLET_BUTTON.TYPES.map(type => <WalletButton {...{ type, key: type }}/>
```

Reply:
How do I export the constants?  is it `export default TYPES`?

Reply:
this passed linter test, but I'm not sure if it's correct yet:
`export default {
  TYPES,
  LINK,
  ICON,
  COLOR,
}`

Reply:
figured out that I had to require JSX and React in the const file in order to get icons to pass through.

Reply:
type styling works now.  used `[square brackets]`, they flatten out everything inside allowing complex compilation styling.

Reply:
incorporated all of this feedback into the component render method.